### PR TITLE
Minor changes found during review

### PR DESCRIPTION
**Pull request by omus**
I was reading over Arbiter.jl and found some very minor issues that I figured I would just correct.

### CODE REVIEW

File: src/graph.jl [hunk 1]
```diff
@@ -82,7 +82,7 @@ visited: (optional, None) If given, a set of nodes that have
 
 NOTE: If node doesn't exist, the method will return False.
 """
-function ancestor_of(graph::ArbiterGraph, name::Symbol, ancestor::Symbol, visited::NodeSet)
+function ancestor_of(graph::ArbiterGraph, name::Symbol, ancestor::Symbol, visited::NodeSet=NodeSet())
```
Comment (iamed2) [inline src/graph.jl]: This will only create NodeSet() once and save it with the method. This is probably not what you want.

Comment (omus) [inline src/graph.jl]: In this context that seems completely reasonable.

**Comment by omus** [comment]
@iamed2 can we get this merged?

File: src/graph.jl (hunk 1 above)
Comment (iamed2) [inline src/graph.jl]: @omus I don't understand how this is reasonable. Seems like we just don't have tests which uncover the bug. NodeSet() is mutable and is mutated in the function. That will affect behaviour of future calls to this function.

Comment (omus) [inline src/graph.jl]: I'll revert that change. When I said it was reasonable I thought that it wasn't being modified within the function.

**Pull request merged**